LICENSE

```diff
@@ -1,13 +1,13 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Financial Times
+Copyright (c) 2015 Financial Times
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+furnished to do so, section to the following conditions:
```

Comment:
haha - that one technically wasn't me, although I did copy the Licence from sections-rw!